scripts/asm-installer/asm_vm

```diff
@@ -1111,6 +1114,21 @@ this project, run:
 EOF
     fi
 
+    info "Verifying the GCE image..."
+    SOURCE_IMAGE_PATH="$(retry 2 run gcloud compute instance-templates \
+      describe "${SOURCE_INSTANCE_TEMPLATE}" \
+    --format=json \
+    --project="${PROJECT_ID}" \
+    | jq '.properties.disks[0].initializeParams.sourceImage')"
+
+    if [[ "${SOURCE_IMAGE_PATH}" == *projects/*-cloud/global/images* ]]; then
+    # check whether the public image is supported
+      if check_not_supported_vm_linux_distro ; then
+        fatal "Image not supported. Supported image OS list are: ${SUPPORTED_VM_DISTROS[*]}"
```

Comment:
nit: Make sure the error message is grammatically correct. e.g. Supported OS distros are: ....